css/style.css

```diff
@@ -0,0 +1,3111 @@
+    width: 100%;
+  }
+
+  .characteristic__carImage {
+    max-width: 90%;
+    padding-top: 40px;
+  }
+
+  .characteristic__description {
+    width: 320px;
+    padding: 18px 40px 43px;
+    display: flex;
+    box-sizing: border-box;
+    flex-direction: row;
+  }
+
+  .description-characteristic__speedo {
+    padding-right: 20px;
+  }
+
+  .speedo__image {
+    width: 38.14px;
+    height: 27.98px;
+  }
+
+  .speedo__title {
+    margin-top: -15px;
+  }
+
+  .speedo__title-numb {
+    font-size: 27px;
+  }
+
+  .speedo__title-text {
+    font-size: 15px;
+  }
+
+  .speedo__desc {
+    font-size: 10px;
+  }
+
+  .description-characteristic__road {
+    padding-right: 0;
+  }
+
+  .road__image-stroke {
+    width: 24.04px;
+    height: 36.88px;
+    padding-top: 10px;
+  }
+
+  .road__image-strokeDotted {
+    width: 21.89px;
+    height: 34.74px;
+    top: 18px;
+  }
+
+  .road__image-ellipseRightTop {
+    width: 5.8px;
+    height: 5.74px;
+    top: 16px;
+  }
+
+  .road__image-ellipseRightTopIn {
+    display: none;
+  }
+
+  .road__image-ellipseLeftBottom {
+    top: 49px;
+  }
+
+  .road__title-numb {
+    font-size: 27px;
+    padding-top: 13px;
+  }
+
+  .road__title-text {
+    font-size: 15px;
+  }
+
+  .road__desc {
+    font-size: 10px;
+  }
+
+  .description-characteristic__testDrive {
+    display: none;
+  }
+
+  .description-characteristic__button {
+    display: none;
+  }
+
+  .description-characteristic__wrap-phone {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    box-sizing: border-box;
+  }
+
+  .description-characteristic__testDrive-phone {
+    display: flex;
+    align-items: center;
+    padding-right: 5px;
+  }
+  
+  .testDrive-phone__image {
+    position: relative;
+    padding-right: 40px;
+  }
+  
+  .testDrive-phone__image-ellipse {
+    position: absolute;
+    width: 22px;
+    height: 22px;
+    top: -11px;
+    left: 10px;
+  }
+  
+  .testDrive-phone__image-tesla {
+    position: absolute;
+    width: 14.18px;
+    height: 8.23px;
+    left: 14px;
+    top: -1px;
+    animation-duration: 3s;
+    animation-name: rotatePhone;
+    animation-iteration-count: infinite;
+    animation-direction: alternate;
+    transform-origin: 50% 0%;
+  }
+  
+  @keyframes rotatePhone {
+    from {
+      transform: rotate(0deg);
+    }
+  
+    to {
+      transform: rotate(-230deg);
+    }
+  }
+  
+  .testDrive-phone__image-dott {
+    position: absolute;
+    width: 2.58px;
+    height: 2.58px;
+    left: 20px;
+    top: -3px;
+  }
+  
+  .testDrive__title-phone {
+    margin: 0px;
+    padding-left: 0px;
+    font-size: 13px;
+    font-family: "MontserratBold";
+    color: #E51937;
+  }
+  
+  .description-characteristic__button-phone {
+    margin: 0px;
+    color: white;
+    background-color: #E51937;
+    display: flex;
+    align-self: center;
+    box-sizing: border-box;
+    font-family: "MontserratBold";
+    font-size: 13px;
+    border: 2px solid #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5558864)